ansible/tasks/additional_storage.yml

```diff
@@ -15,7 +15,9 @@
 - name: "Load a variable files for environment: {{ env }}"
   include_vars: "{{ item }}"
   with_first_found:
-    - "vars/aeternity/{{ env }}.yml"
+    - files:
+        - "vars/aeternity/{{ env }}.yml"
+      skip: true
```

Comment:
Does this skip only this step ("Load a variable files for environment ...") or also the rest of the file?

Reply:
From the error message `fatal: [default]: FAILED! => {"msg": "No file was found when using with_first_found. Use the 'skip: true' option to allow this task to be skipped if no files are found"}`
This just skips this step and continue with the other tasks and try to resolve the variables by other means